docs/reference/rest-api/security/grant-api-keys.asciidoc

```diff
@@ -83,15 +86,28 @@ It supports nested data structure.
 Within the `metadata` object, keys beginning with `_` are reserved for
 system usage.
 
+`client_authentication`::
+(Optional, object) When using the `access_token` grant type, and when supplying a
+JWT, this specifies the client authentication for <<JWT access tokens, jwt-realm-oauth2>>
+ (i.e. what's normally specified by the `ES-Client-Authentication` request header).
+
+`scheme`:::
+(Required, string) The scheme (case-sensitive) as it's supplied in the
+`ES-Client-Authentication` request header.
+
+`value`:::
+(Required, string) The value that follows the scheme for the client credentials
+as it's supplied in the `ES-Client-Authentication` request header.
```

Comment:
Can we add an example ? for example, The header is "ES-Client-Authentication: SharedSecret myShar3dS3cret" then the value here should be "myShar3dS3cret"